src/reduce.test.js

```diff
@@ -11,9 +11,42 @@
     delete Array.prototype.reduce2;
   });
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(Array.prototype.reduce2).toBeInstanceOf(Function);
+  });
+
+  it(`should be correctly if 'startValue' === 0`, () => {
+    const result = [1, 2, 3].reduce2((acc, num) => acc + num, 0);
+
+    expect(result).toBe(6);
+  });
+
+  it(`should be correctly if 'startValue' > 0`, () => {
+    const result = [1, 2, 3].reduce2((acc, num) => acc + num, 6);
+
+    expect(result).toBe(12);
+  });
 
+  it(`should be correctly if without 'startValue'`, () => {
+    const result = [1, 2, 3].reduce2((acc, num) => acc + num);
+
+    expect(result).toBe(6);
+  });
+
+  it(`should be correctly if string array`, () => {
```

Comment:
The test description `should be correctly if string array` could be improved for clarity. Consider rephrasing it to something like `should concatenate strings correctly when reducing a string array`.